src/log/genode/gneiss-log.adb

```diff
@@ -25,4 +25,6 @@ is
    function Index (Session : Dispatcher_Session) return Session_Index_Option is
       (Session.Index);
 
+   function Registered (Session : Dispatcher_Session) return Boolean is (True);
+
```

Comment:
This is a pure proof function. Since we do not proof on Genode and it is never actually checked I set it to `True`. Maybe setting it to `False` would be a better default. For an actual implementation we would have to set a dedicated flag that is never actually checked.

Reply:
We should set it to `False`. Also, we should prefix it with `Prf_` to clarify its purpose.

Reply:
Do we want to have functions prefixed with `Prf_` as part of the public API?

Reply:
Thinking about it again, we don't. It's `Ghost` and the compiler will tell you when using it outside proof context.

Reply:
Fixed in https://github.com/Componolit/gneiss/pull/173/commits/aaa74c5dcd1170799b197cfa7279b62ce58baa4d.